### ISSUE TITLE
Possible to have state report on/off instead of event name?

### ISSUE THREAD

**Issue by joaldes** [tybritten/ical-sensor-homeassistant#58]
The state for the events is the name of the event and the date/time it starts. Is it possible to change it to show on or off depending on if the event has started?

**Comment by Olen** [comment]
Can't you use the "calendar.ical_xxxx" for that?

**Comment by joaldes** [comment]
Oh, I didn't realize that entity worked in that fashion.  Is it always related to the next upcoming event?

**Comment by Olen** [comment]
Yes.  The calendar entity state is always related to the next (or current) event.
It works the same as e.g. https://www.home-assistant.io/integrations/google/#using-calendar-in-automations
